plugin/kubectl-check_acr

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+
+context=""
+kubeconfig=""
+generator=""
+acr=""
+namespace=""
+while [ $# -gt 0 ]; do
+  key="$1"
+
+  case $key in
+  --context)
+    context="--context $2"
+    shift
+    shift
+    ;;
+  --kubeconfig)
+    kubeconfig="--kubeconfig $2"
+    shift
+    shift
+    ;;
+  -n | --namespace)
+    namespace="--namespace $2"
+    shift
+    shift
+    ;;
+  *)
+    acr="$1"
+    shift
+    ;;
+  esac
+done
+
+if [ -z "$acr" ]; then
+  echo "Please specify ACR name"
+  exit 1
+fi
+
+image="fseldow/canipull"
+pod="canipull-$(env LC_ALL=C tr -dc a-z0-9 < /dev/urandom | head -c 6)"
+
+overrides="$(
+  cat <<EOT
+{
+  "spec": {
+    "restartPolicy": "Never",
+    "hostNetwork": true,
+    "containers": [
+      {
+        "securityContext": {
+          "runAsUser": 0
+        },
+        "image": "$image",
+        "name": "canipull",
+        "stdin": true,
+        "stdinOnce": true,
+        "tty": true,
+        "args": [ "-v6", "$acr" ],
+        "volumeMounts": [
+          {
+            "name": "azurejson",
+            "mountPath": "/etc/kubernetes"
+          },
+          {
+            "name": "sslcerts",
+            "mountPath": "/etc/ssl/certs"
+          }
+        ]
+      }
+    ],
+    "tolerations": [
+      {
+        "key": "CriticalAddonsOnly",
+        "operator": "Exists"
+      },
+      {
+        "effect": "NoExecute",
+        "operator": "Exists"
+      }
+    ],
+    "volumes": [
+      {
+        "name": "azurejson",
+        "hostPath": {
+          "path": "/etc/kubernetes"
+        }
+      },
+      {
+        "name": "sslcerts",
+        "hostPath": {
+          "path": "/etc/ssl/certs"
+        }
+      }
+    ]
+  }
+}
+EOT
+)"
+
+# Support Kubectl <1.18
```

Comment:
if this is a test tool no need such

Reply:
maybe add this into a doc folder as a dev intro tool?

Reply:
rename repo as doc